cplusplus.json

```diff
@@ -25,5 +25,119 @@
+  "imports": ["gflags/gflags.h"], "examples": ["https://gflags.github.io/gflags/"]},
+  {"id": "cpp.ncurses", "name": "Ncurses", "tech": ["api"], "tags": ["gui", "framework"], "repo": "http://invisible-island.net/ncurses/",
+  "imports": ["ncurses_cfg.h", "curses.h"], "examples": ["http://invisible-island.net/ncurses/ncurses-examples.html"]},
+  {"id": "cpp.pdcurses", "name": "PDCurses", "tech": ["api"], "tags": ["gui", "framework"], "repo": "wmcbrine/PDCurses",
+  "imports": ["curses.h"], "examples": ["https://github.com/wmcbrine/PDCurses/tree/master/demos"]},
+  {"id": "cpp.replxx", "name": "replxx", "tech": ["cli"], "tags": ["readline", "interpreter"], "repo": "AmokHuginnsson/replxx",
+  "imports": ["replxx.hxx"], "examples": ["https://github.com/AmokHuginnsson/replxx/tree/master/examples"]},
+  {"id": "cpp.tclap", "name": "TCLAP", "tech": ["utils"], "tags": ["argument-parsing"], "repo": "http://tclap.sourceforge.net/",
+  "imports": ["tclap/CmdLine.h"], "examples": ["https://github.com/eile/tclap/tree/master/examples"]},
+  {"id": "cpp.termbox", "name": "Termbox", "tech": ["api"], "tags": ["gui", "framework"], "repo": "nsf/termbox",
+  "imports": ["termbox.h"], "examples": ["https://github.com/nsf/termbox/tree/master/src/demo"]},
+
+  {"id": "cpp.alpaka", "name": "alpaka", "tech": ["faster-code"], "tags": ["concurrency", "high-performance"], "repo": "ComputationalRadiationPhysics/alpaka",
+  "imports": ["alpaka/alpaka.hp"], "examples": ["https://github.com/ComputationalRadiationPhysics/alpaka/tree/develop/example"]},
+  {"id": "cpp.arrayfire", "name": "ArrayFire", "tech": ["faster-code"], "tags": ["concurrency", "high-performance"], "repo": "arrayfire/arrayfire",
+  "imports": ["arrayfire.h"], "examples": ["https://github.com/arrayfire/arrayfire/tree/master/examples"]},
+  {"id": "cpp.boost-compute", "name": "Boost.Compute", "tech": ["faster-code"], "tags": ["concurrency", "high-performance", "gpu"], "repo": "boostorg/compute",
+  "imports": ["boost/compute/core.hpp"], "examples": ["https://github.com/boostorg/compute/tree/master/example"]},
+  {"id": "cpp.bolt", "name": "Bolt", "tech": ["faster-code"], "tags": ["concurrency", "high-performance", "gpu"], "repo": "HSA-Libraries/Bolt",
+  "imports": ["bolt/unicode.h", "bolt/cl/sort.h", "bolt/cl/reduce.h"], "examples": ["https://github.com/HSA-Libraries/Bolt/tree/master/examples"]},
+  {"id": "cpp.ck", "name": "ck", "tech": ["data-structures"], "tags": ["high-performance"], "repo": "concurrencykit/ck",
+  "imports": ["ck_array.h"], "examples": []},
+  {"id": "cpp.concurrentqueue", "name": "ConcurrentQueue", "tech": ["data-structures"], "tags": ["high-performance"], "repo": "cameron314/concurrentqueue",
+  "imports": ["blockingconcurrentqueue.h", "concurrentqueue.h"], "examples": ["https://github.com/cameron314/concurrentqueue/blob/master/samples.md"]},
+  {"id": "cpp.cub", "name": "CUB","tech": ["faster-code"], "tags": ["concurrency", "high-performance", "cuda"], "repo": "NVlabs/cub",
```

Comment:
let's make tags == cuda
i think that 'concurrency' is not descriptive enough, and as a rule it makes sense to highlight gpu stuff.